examples/form_demo/local/lib/form_demo_local.ex

```diff
@@ -66,6 +67,37 @@ defmodule FormDemoLocal do
     end
   end
 
+  def handle_event(
```

Comment:
Maybe you could create a new callback for server messages? Like `handle_server`?